src/test/java/Homework_8/Homework_8.java

```diff
@@ -0,0 +1,49 @@
+package Homework_8;
+
+import BaseObjects.BaseTest;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.Select;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Homework_8 extends BaseTest {
+
+    @Test(priority = 1)
+    public void checkFirstValueInList() {
+        driver.get("D:/AutomationPesetskaya_QA11/src/test/java/Homework_8/task_8.html");
+        //Создать коллекцию из значений в первой колонке
+        List<WebElement> brandsWebEl = driver.findElements(By.xpath("//tr/td[1]"));
```

Comment:
Prettier locator for table element would be By.cssSelector(td:first-child)

Reply:
It's just a note

Reply:
Thanks for advice - already changed